arch/x86/crc32_fold_pclmulqdq.c

```diff
@@ -17,19 +17,20 @@
  */
 
 #ifdef X86_PCLMULQDQ_CRC
+#include "../../zutil.h"
```

Comment:
Relative paths should be unnecessary as root of source tree is included in search path.

Reply:
If I remember correctly, there was a PR some time ago that converted everything to use relative paths. If you do a search for "../../" in code base there is a lot of instances.

Reply:
I can't veto "pointless" pull requests, because I'm not the project owner... If one of my recent pull request gets commented because I used relative paths, then obviously there is no consensus about using or not using them.

We all have other projects besides zlib-ng, so going through every single pull request or commit would exhaust our spare time.

I don't like voting down pull requests, because then the author just finds other people to vote it up instead, but there is at most handful of pull requests where I've said clearly that I think the pull request is not the best way to do things.

If there is currently three long-time contributors, it will result in one downvote and two upvotes, which defeats the purpose of downvoting.

Reply:
I am not advocating one way or another. I don't mind either way. But I just want to be consistent with what we already have. I wouldn't mind a PR that went a cleaned them up.

However, I did find the PR that made the change - https://github.com/zlib-ng/zlib-ng/pull/383. Some may argue that the use case mentioned in that PR is not important. @Dead2 might be able to comment about why he thought it was necessary.

Reply:
I bought this current development PC in November 2019, so that was merged in after my previous development broke in early 2019, so I wasn't even online to comment.

Reply:
I actually have no idea what caused me to think that was a good change..

It makes very little sense to me in retrospect, but it looks like it made sense back then, for whatever reason. Probably related to being more compatible with stock zlib, or inclusion into external build system, but with functable and the need for different compiler flags for separate compilation units, I don't think that is a good idea.

I would be fine with merging a PR that reverts that if there is a consensus that it would be cleaner.